### PR TITLE
LPS-113441 BuildLang

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/service/impl/RedirectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -407,6 +409,24 @@ private void _validate(String destinationURL, String sourceURL)
 		if (Validator.isNull(sourceURL)) {
 			throw new RequiredRedirectEntrySourceURLException();
 		}
+
+		_validateURL(sourceURL);
+	}
+
+	private void _validateURL(String sourceURL)
+		throws LayoutFriendlyURLException {
```
Comment (4lejandrito) [inline modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/service/impl/RedirectEntryLocalServiceImpl.java:417]: I am not sure if we should use Layout exceptions here. We can discuss it tomorrow.

**Comment by 4lejandrito** [comment]
Hi @AliciaGarciaGarcia,

We are moving LayoutFriendlyURL logic to the friendly URL service. This means at some point we will get rid of LayoutFriendlyURLs. 

We should try to do this validation using the friendly url service if possible. We can have a look together tomorrow.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by yichenroy** [comment]
ci:forward

There was an outage related to GitHub api calls.

**Pull request closed without merging**